src/components/DockerImageSelect.vue

```diff
@@ -9,6 +18,12 @@ import store from "@/store";
 import propertiesStore from "@/store/properties";
 import { IWorkerLabels } from "@/store/model";
 
+interface DockerImageSelectEntry {
```

Comment:
Can you use `IDockerImageSelectEntry` instead please?
The `I` at the beginning of the interface is an implicit style guideline (I admit it is questionable)